contrib/leptonica/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "leptonica"
+pkgver = "1.84.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DBUILD_SHARED_LIBS=ON", "-DCMAKE_BUILD_TYPE=Release"]
```

Comment:
```suggestion
configure_args = ["-DBUILD_SHARED_LIBS=ON"]
```

doesn't need to be set here seemingly